e2e/process-services-cloud/process-list-selection-cloud.e2e.ts

```diff
@@ -0,0 +1,138 @@
+            tasksCloudDemoPage.clickSettingsButton().selectSelectionMode('Multiple');
+            tasksCloudDemoPage.clickAppButton();
+            expect(processCloudDemoPage.getActiveFilterName()).toEqual('Running Processes');
+
+            processCloudDemoPage.processListCloudComponent().getDataTable().selectRowByRowName(processInstances[0]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsSelectedByName(processInstances[0]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().selectRowByNameWithKeyboard(processInstances[1]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsSelectedByName(processInstances[0]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsSelectedByName(processInstances[1]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsNotSelectedByName(processInstances[2]);
+        });
+
+        it('[C297465] Should be able to select multiple processes using checkboxes', () => {
+            tasksCloudDemoPage.clickSettingsButton().enableMultiSelection();
+            tasksCloudDemoPage.clickAppButton();
+            expect(processCloudDemoPage.getActiveFilterName()).toEqual('Running Processes');
+
+            processCloudDemoPage.processListCloudComponent().getDataTable().clickCheckboxByName(processInstances[0]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsCheckedByName(processInstances[0]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().clickCheckboxByName(processInstances[1]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsCheckedByName(processInstances[1]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsNotCheckedByName(processInstances[2]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().clickCheckboxByName(processInstances[1]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsNotCheckedByName(processInstances[1]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsCheckedByName(processInstances[0]);
+        });
+
+        it('[C299125] Should be possible select all the rows when multiselect is true', () => {
+            tasksCloudDemoPage.clickSettingsButton().enableMultiSelection();
+            tasksCloudDemoPage.clickAppButton();
+            expect(processCloudDemoPage.getActiveFilterName()).toEqual('Running Processes');
+
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkAllRowsButtonIsDisplayed().checkAllRows();
+            processCloudDemoPage.processListCloudComponent().getDataTable().checkRowIsCheckedByName(processInstances[0]);
+            processCloudDemoPage.processListCloudComponent().getDataTable().clickCheckboxByName(processInstances[1]);
```

Comment:
I think, the above line needs to have checkRowIsCheckedByName(processInstances[1])